src/app/controllers/admin-forms.server.controller.js

```diff
@@ -8,7 +8,6 @@ const moment = require('moment-timezone')
 const _ = require('lodash')
 const JSONStream = require('JSONStream')
 const { StatusCodes } = require('http-status-codes')
-const get = require('lodash/get')
```

Comment:
how did you manage to commit this though? did the linter not catch it?

Reply:
It may have snuck in while I was rebasing, which isn't covered by the pre-commit hooks

Reply:
We could add a `post-rewrite` hook to catch rebase and commit --amend but I discussed with @karrui and it may be annoying to trigger the hooks every time there's a rebase

Reply:
aha, makes sense